packages/decoupled-kit-health-check/__tests__/__snapshots__/GatsbyWordPressHealthCheck.test.ts.snap

```
@@ -1,5 +1,14 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`GatsbyWordPressHealthCheck > should fail if no protocol is set for the WPGRAPHQL_URL 1`] = `
+[
+  "No .env* file found, assuming production environment.",
+  "Checking for PANTHEON_CMS_ENDPOINT or WPGRAPHQL_URL...",
+  "⎩✅ WPGRAPHQL_URL is set!",
+  "Validating CMS endpoint...",
```

We should throw an error in this case, probably `InvalidCMSEndpointError`